tests/snapshots/test_cli__long-help.snap

```diff
@@ -3,17 +3,47 @@ source: tests/test-cli.rs
 expression: format_outputs(&output)
 ---
 stdout:
-cargo-vet 0.3.0
+cargo-vet-vet 0.3.0
```

Comment:
This is a regression, unclear whose fault it is (see https://github.com/clap-rs/clap/issues/4132#issuecomment-1229101979).

Reply:
If you set `name = cargo` in addition to the bin_name, this should be resolved.  See https://github.com/clap-rs/clap/blob/master/examples/cargo-example-derive.rs#L4

In clap 3, the user sets `bin_name = cargo` and we build that up as we recurse down the subcommands.  When we then render the "name", we then did a search/replace on it which on windows without an explicit `bin_name` would end up with `foo.exe-bar`.  So in clap 4, we build up the `usage_name` from the root command's `bin_name` + each subcommand's `name` and the `display_name` from all command's `name`s.